src/Tuque/IslandoraFedoraRepository.php

```diff
@@ -99,7 +99,10 @@ public function getNextIdentifier($namespace = NULL, $create_uuid = NULL, $count
    *   'islandora_basic_collection_generate_uuid' Drupal variable; otherwise,
    *   the value of $to_create itself.
    */
+  // @codingStandardsIgnoreStart
   protected static function useUUIDs($to_create) {
+  // Rename would break older functionality.   
```

Comment:
I'm... not sure what this is supposed to indicate.

Reply:
https://github.com/discoverygarden/islandora/pull/5/files/e13d209e855250a894acbf81a7d14da341b46cc8#r153370997

Will asked me to revert Camel case because it would break d7 apis.

Reply:
In what way? As already discussed, function (and method) names are not case sensitive.

... That said, it may make sense to make sure that any calls follow the same case structure, just for consistencies sake.

Reply:
@adam-vessey @willtp87 Can I get a consensus on this? Change or no-change?

Reply:
@adam-vessey 's analysis stands.